lib/common/utils/notification_service.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:awesome_notifications/awesome_notifications.dart';
+import 'package:flutter/material.dart';
+import 'package:fluttercon/core/theme/theme_colors.dart';
+
+class NotificationService {
```

Comment:
Please register this as a singleton with Injectable.

You can do this by adding the `@singleton` tag on this class. Ref: `lib/common/repository/auth_repository.dart:8` so that you can use it with `get_it`. You will then initialise this class like so:

`getIt<NotificationService>().initNotifications();`